mayhem/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
```

Comment:
Not sure we need an entire build script for this? This could easily be placed in the Dockerfile.

Reply:
I'm using the build script for standardization purposes, but I can see how it would be seen as unnecessary. I'll remove it.

Reply:
Also, as far as I know, the base image is actually providing some utilities here, simply moving this call into docker file did not work for me. I do not recall exactly what is happening, I would have to read the docs again.